comp/core/tagger/taggerimpl/collectors/workloadmeta_extract.go

```diff
@@ -650,6 +676,17 @@ func (c *WorkloadMetaCollector) extractTagsFromPodContainer(pod *workloadmeta.Ku
 	// enrich with standard tags from environment variables
 	c.extractFromMapWithFn(container.EnvVars, standardEnvKeys, tags.AddStandard)
 
+	// standard tags in OpenTelemetry SDK format from environment
```

Comment:
This block of code is the same as the above one. Consider extracting to a function.

Reply:
Done 👍